src/Tools/dotnet-getdocument/src/Exe.cs

```diff
@@ -32,100 +34,30 @@ internal static class Exe
                 startInfo.WorkingDirectory = workingDirectory;
             }
 
-            using (var process = Process.Start(startInfo))
+            using var process = Process.Start(startInfo);
```

Comment:
Why change to use this variant of `using`? Having the block level using makes it clear when you expect the process has exited.

Reply:
Compiler goodies. Remove the bracket will help in defining the terse syntax for using scope and that it is pinned to func.